agents/TransactionAgent.java

```diff
@@ -72,8 +74,12 @@ private void insertTransaction(Pair<Long, Double> transaction, Long sellerCompan
                                 .rel("transaction_merchandise", Graql.var("product"))
 //                                .has("currency")  // TODO Add currency
```

Comment:
Do we not have currency available to complete this attribute? If not, and completing this part requires some other larger change, it would be good to create an issue now for that other change and link to it here.